src/MessageDialog.Uno.WinUI/Extensions/DispatcherQueueExtensions.cs

```diff
@@ -0,0 +1,94 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
```

Comment:
This repository (like all our repos) is under the Apache-2 license.
I'm not sure this header is right.